doc/release/upcoming_changes/14877. new_feature.rst

```diff
@@ -0,0 +1,4 @@
+`numpy.unwrap` has new `min_val` and `max_val` arguments for handling non-radian wrapping
```

Comment:
```suggestion
`numpy.unwrap` has new ``min_val`` and ``max_val`` arguments for handling non-radian wrapping
```

Double backticks for code, single for function names that can be linked. Will need to lengthen the underline too.